template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/_LoginPartial.cshtml

```diff
@@ -0,0 +1,68 @@
+@*#if (IndividualB2CAuth)
+@using Microsoft.AspNetCore.Authentication.Extensions
+@using Microsoft.Extensions.Options
+
+#endif
+@*#if (IndividualB2CAuth)
+@inject IOptions<AzureAdB2COptions> AzureAdB2COptions
+
+#endif *@
+@*#if (IndividualLocalAuth) *@
+@using Microsoft.AspNetCore.Identity
+@using Company.WebApplication1.Data
+@*#else
+@using System.Security.Principal
+#endif *@
+
+@*#if (IndividualLocalAuth) *@
+@inject SignInManager<ApplicationUser> SignInManager
+@inject UserManager<ApplicationUser> UserManager
+@*#endif *@
+@*#if (IndividualLocalAuth) *@
```

Comment:
Like above, this @*#if is conditioning on the same thing as the previous block which ends on the previous line - could remove line 20 & 21.